service/internal/proctelemetry/config.go

```diff
@@ -101,8 +103,9 @@ func InitPrometheusServer(registry *prometheus.Registry, address string, asyncEr
 	mux := http.NewServeMux()
 	mux.Handle("/metrics", promhttp.HandlerFor(registry, promhttp.HandlerOpts{}))
 	server := &http.Server{
-		Addr:    address,
-		Handler: mux,
+		Addr:              address,
+		Handler:           mux,
+		ReadHeaderTimeout: defaultReadHeaderTimeout,
```

Comment:
Just to double-check. this is unrelated to the change above, right?

Reply:
it's because I also removed the gosec exception. The linter picked up the fact we didn't set this on the prometheus server as well.